src/App.js

```diff
@@ -13,19 +13,57 @@ import Me from "./Views/Me/Me";
 import MeBooks from "./Views/Me/MeBooks";
 import MeBaskets from "./Views/Me/MeBaskets";
 import MeConversations from "./Views/Me/MeConversations";
-import MePokes from "./Views/Me/MePokes";
+import Pokes from "./Views/Pokes/Pokes";
 import Basket from "./Views/Basket/Basket";
 import Error from "./Views/Error";
 
+import fetchPokes from "./API/fetchPokes";
+import { fetchLoggedUser } from "./API/fetchUser";
+
 function App() {
   const [accessToken, setAccessToken] = useState("");
+  const [loggedUser, setLoggedUser] = useState("");
```

Comment:
`loggedUser` is a string? So maybe better name would be `loggedUserName` or something? `loggedUser` suggests some kind of object with fields.
What do you think?